sklearn/datasets/samples_generator.py

```diff
@@ -61,7 +62,8 @@ def make_classification(n_samples=100, n_features=20, n_informative=2,
         The total number of features. These comprise `n_informative`
         informative features, `n_redundant` redundant features, `n_repeated`
         duplicated features and `n_features-n_informative-n_redundant-
-        n_repeated` useless features drawn at random.
+        n_repeated` useless features drawn at random.  A larger number of
+        features generally makes the classification task more difficult.
```

Comment:
Are you sure about that?

Reply:
Was just thinking curse of dimensionality and all, but I can remove it.